src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,265 @@
+                # replace multiple values by single value
+                copied_row[column_index] = item
+                # add expanded row to array
+                values_data = np.concatenate((values_data, copied_row), axis=0)
+
+    df = pd.DataFrame(values_data,
+                      index=[x for x in range(len(values_data))],
+                      columns=[SOURCE, TARGET, ALT_SOURCE_ID, ALT_TARGET_ID, RELATION, PUBMED_ID],
+                      )
+
+    df = df.drop(df[df.ALT_SOURCE_ID] == list)
+
+    return df
+
+
+def get_processed_biogrid() -> pd.DataFrame:
+    """Load BioGRDID file, filter and rename columns and return a dataframe.
+
+    :return: dataframe of preprocessed BioGRID data
+    """
+    df = _get_my_df()
+    # rename columns
+    df = df.rename(columns=BIOGRID_COLUMN_MAPPER)
+
+    # take relevant columns for source, target, alternative ids, relation and PubMed ID
+    df = df[[SOURCE, TARGET, ALT_SOURCE_ID, ALT_TARGET_ID, RELATION, PUBMED_ID]]
+
+    # filter for uniprot
+    df[SOURCE] = filter_for_prefix_single(list_ids=df[SOURCE], prefix=UNIPROT)
+    df[TARGET] = filter_for_prefix_single(list_ids=df[TARGET], prefix=UNIPROT)
+    # also in alternative ids
+    df[ALT_SOURCE_ID] = filter_for_prefix_multi(list_ids=df[ALT_SOURCE_ID], prefix=UNIPROT)
+    df[ALT_TARGET_ID] = filter_for_prefix_multi(list_ids=df[ALT_TARGET_ID], prefix=UNIPROT)
+
+    # drop rows if no uniprot id
+    df = df.dropna(subset=[SOURCE, ALT_SOURCE_ID], how='all')
+    df = df.dropna(subset=[TARGET, ALT_TARGET_ID], how='all')
+
+    # expand dataframe if multiple uniprot ids exist
+    df = expand_df(df=df, column_name=ALT_SOURCE_ID)
+    df = expand_df(df=df, column_name=ALT_TARGET_ID)
+
+    # filter for relation
+    df[RELATION] = filter_for_prefix_single(
+        list_ids=df[RELATION],
+        rstrip=')',
+        lstrip='(',
+        separator='"',
+        prefix='(',
+    )
+
+    return df
+
+
+def get_bel() -> BELGraph:
+    """Get a BEL graph for IntAct.
+
+    :return: BEL graph
+    """
+    df = get_processed_biogrid()
+    graph = BELGraph(name=BIOGRID)
+    for _, row in tqdm(df.iterrows(), total=len(df.index), desc=f'mapping {BIOGRID}'):
+        _add_my_row(graph, row)
+    return graph
+
+
+def _add_my_row(graph: BELGraph, row) -> None:  # noqa:C901
+    """Add for every pubmed ID an edge with information about relationship type, source and target.
+
+    :param graph: graph to add edges to
+    :param row: row with metainformation about source, target, relation, pubmed_id
+    :return: None
+    """
+    relation = row[RELATION]
+    source_uniprot_id = row[SOURCE]
+    target_uniprot_id = row[TARGET]
+
+    pubmed_ids = row[PUBMED_ID]
+
+    source = pybel.dsl.Protein(
+        namespace='uniprot/swiss-prot',
+        identifier=source_uniprot_id,
+        name=get_mnemonic(source_uniprot_id),
+    )
+    target = pybel.dsl.Protein(
+        namespace='uniprot/swiss-prot',
```

Comment:
just uniprot

Reply:
Tried to resolve it with this expression here: c7734ef8f5969250af21500444c99c23eb5472bb

But it didn't work for some reason. Would you know why?